pkg/scripts/kubeadm.go

```diff
@@ -44,7 +44,7 @@ var (
 			exit 0;
 		fi
 
-		sudo kubeadm {{ .VERBOSE }} init --config={{ .WORK_DIR }}/cfg/master_{{ .NODE_ID }}.yaml
+		sudo kubeadm {{ .VERBOSE }} init --skip-phases={{ .SKIP_PHASE }} --config={{ .WORK_DIR }}/cfg/master_{{ .NODE_ID }}.yaml
```

Comment:
I think we should skip providing this flag if it's empty to be on the safe side if kubeadm changes how's this flag handled someday (e.g. if they change the default value). The simplest way would be just to check if this field is not empty, although, there might be more elegant ways as well. I think we can go with the simplest way for now.

```suggestion
		sudo kubeadm {{ .VERBOSE }} init {{ if .SKIP_PHASE }}--skip-phases={{ .SKIP_PHASE }}{{ end}} --config={{ .WORK_DIR }}/cfg/master_{{ .NODE_ID }}.yaml
```